### PR TITLE
Merged My Books and user menu

### LINKED ISSUE #1514: Should My books and user menus be merged?
screen shot 2018-11-04 at 8 14 18 am
screen shot 2018-11-04 at 8 14 22 am

Should the "my reading lists" menu be moved into the user menu?

If that happens, these menus have identical contents.

Given they take up valuable screen real estate on mobile it doesn't really make sense to have them both... what am I missing?

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #1514 

Merged my books and user menu. 

### Screenshot

image

### Stakeholders

@jdlrobson

### CODE REVIEW

**Comment by Yashs911** [comment]
@jdlrobson Merged my books and user menu.
Additionally, I replaced the "Help Center" with "Help & Support" will it be okay or revert it.

File: static/css/components/header-bar.less [hunk 1]
```diff
@@ -48,7 +48,7 @@
       position: absolute;
       z-index: @z-index-level-5;
       text-align: right;
-      width: 125px;
+      width: 179px;
```
Comment (jdlrobson) [inline static/css/components/header-bar.less:51]: It should be possible for this menu to work without defining an explicit width.
This width is going to cause us problems later on as we i18n the site.

Can you try using `white-space: nowrap;` on the link instead?

File: openlibrary/templates/lib/nav_foot.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h2>$:_('Develop')</h2>
       <div>
         <h2>$:_('Help')</h2>
         <ul>
-          <li><a href="/help" title="$_('Help Center')">$_('Help Center')</a></li>
+          <li><a href="/help" title="$_('Help & Support')">$_('Help & Support')</a></li>
```
Comment (jdlrobson) [inline openlibrary/templates/lib/nav_foot.html:41]: this should be done in a separate change. This is scope creep and that makes it harder for deployers if they need to revert functionality.

Comment (Yashs911) [reply openlibrary/templates/lib/nav_foot.html:41]: Okay.

File: static/css/components/header-bar.less (hunk 1 above)
Comment (Yashs911) [reply static/css/components/header-bar.less:51]: @jdlrobson
image

Comment (Yashs911) [reply static/css/components/header-bar.less:51]: @jdlrobson
image

**Comment by Yashs911** [comment]
@jdlrobson I have made the required changes. Should I remove the right and left padding from the list items so that it doesn't occupy much space?

**Comment by jdlrobson** [comment]
I think the padding is fine. Let's keep that.

**Review by jdlrobson — APPROVED** [review]
Code looks good. Will pull down and test tomorrow.

**Comment by jdlrobson** [comment]
Remaining:
1) Please reduce the z-index on the "work-menu sticky" element on http://localhost:8080/works/OL53924W/The_complete_works_of_Mark_Twain:

Screen Shot 2021-01-15 at 11 29 08 AM

* My reading log should be on one line. I'd suggest a min-width is appropriate here - keep the 150px from before.

2) My reading log and my reading stats should be on a single line
Screen Shot 2021-01-15 at 11 32 31 AM

**Comment by Yashs911** [comment]
> 1. My reading log and my reading stats should be on a single line

@jdlrobson I guess you need to `make css` because I added `whitespace: nowrap;` so it should be on one line.

**Comment by Yashs911** [comment]
@jdlrobson reduced z-index.
image

**Comment by jdlrobson** [comment]
Gonna try and take some time away from keyboard this weekend but thanks for the note about make css.  I suspect it's more than likely an issus my end!

**Comment by jdlrobson** [comment]
We should show the more menu now, with the new found space. We don't show the more menu at mobile resolutions, as previously there was no space but now we should.
